set.mm

```diff
@@ -257966,7 +258036,6 @@ is also applicable for semirings (without using the commutativity of the
       VC $.
   $}
 
-
```

Comment:
I think the policy is to have two blank lines before the start of a new chapter, so this line should not be removed.
(not blocking for me)